.github/scripts/markdown-link-check-config.json

```diff
@@ -1,8 +1,13 @@
 {
-  "ignorePatterns": [
+  "retryOn429": true,
+  "httpHeaders": [
     {
-      "pattern": "^https://mvnrepository\\.com/artifact/"
+      "urls": [
+        "https://docs.github.com"
+      ],
+      "headers": {
+        "Authorization": "Bearer {{ env.GITHUB_TOKEN }}"
```

Comment:
BTW any reference for this env pattern? Don't see it in the docs. Found this technique searching on github. Can try these in separate PR of course

https://github.com/ctreffs/SwiftFSOMonitor/blob/541426bdbe073080ad103edde9083350db56016a/.github/workflows/markdown-link-check.yml#L17

Reply:
this is where I found that: https://github.com/tcort/markdown-link-check/pull/146

Reply:
Not sure if I'm missing any regex flag but looks like the spaces around the braces could cause a problem

https://github.com/tcort/markdown-link-check/pull/146/files#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346R10

Reply:
will try 👍